ProductsService/ProductsService.Application/Mapping/Discounts/AddDiscountRequestMappingProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using ProductsService.Application.UseCases.DiscountUseCases.Commands.AddDiscount;
+using ProductsService.Domain.Entities;
+
+namespace ProductsService.Application.Mapping.Discounts;
+
+internal class AddDiscountRequestMappingProfile : Profile
+{
+    public AddDiscountRequestMappingProfile()
+    {
+        CreateMap<AddDiscountRequest, Discount>()
```

Comment:
What if you want to add discount for the future?

Reply:
Added StartDate into AddDiscountRequest